test.js

```diff
@@ -0,0 +1,170 @@
+    return requestWithSupertest
+      .get(
+        `/api/mint?walletAddress=${testUser.classicAddress}&tokenCount=300&url=ipfs://QmQDDD1cNgnyhPC4pBLZKhVeu12oyfCAJoWr1Qc1QgbkPN&title=test_title&desc=test_description&loc=Warsaw`
+      )
+      .then(async (r) => {
+        console.log(JSON.parse(r.text).result);
+        testEvent = await JSON.parse(r.text).result;
+        r.res.statusCode.should.equal(200);
+        JSON.parse(r.text).result.should.be.a("object");
+      });
+  }).timeout(6000000);
+  */
+
+  it("Checking if it's possible to claim NFT for event", async () => {
+    return requestWithSupertest
+      .get(
+        `/api/claim?walletAddress=${testUser.classicAddress}&minter=${minter}&eventId=${testEvent.eventId}&type=1`
+      )
+      .then((r) => {
+        console.log(JSON.parse(r.text).result.length);
+        r.res.statusCode.should.equal(200);
+        // JSON.parse(r.text).result.should.be.a("object");
+        JSON.parse(r.text).result.should.be.a("array");
+        JSON.parse(r.text).status.should.equal("success");
+      });
+  }).timeout(600000);
+
+  it("Claiming offer for NFT from event", async () => {
+    return requestWithSupertest
+      .get(
+        `/api/claim?walletAddress=${testUser.classicAddress}&minter=${minter}&eventId=${testEvent.eventId}&type=2`
+      )
+      .then((r) => {
+        console.log(JSON.parse(r.text).result.length);
+        console.log("offer ", JSON.parse(r.text).offer);
+        r.res.statusCode.should.equal(200);
+        // JSON.parse(r.text).result.should.be.a("object");
+        JSON.parse(r.text).result.should.be.a("array");
+        JSON.parse(r.text).status.should.equal("transferred");
+      });
+  }).timeout(600000);
+
+  it("Looking up if the test user is on the attendees list for test event", async () => {
+    return requestWithSupertest
+      .get(`/api/attendees?minter=${minter}&eventId=${testEvent.eventId}`)
+      .then((r) => {
+        console.log(JSON.parse(r.text));
+        r.res.statusCode.should.equal(200);
+        JSON.parse(r.text).result.should.be.a("array");
+        JSON.parse(r.text).result[0].user.should.equal(testUser.classicAddress);
+      });
+  }).timeout(600000);
+
+  it("Obtaining random ID for verification process", async () => {
+    return requestWithSupertest
+      .get(
+        `/api/startVerification?walletAddress=${walletForSignatureVerification.classicAddress}`
+      )
+      .then(async (r) => {
+        testMemoId = await JSON.parse(r.text).result;
+        console.log(JSON.parse(r.text));
+        r.res.statusCode.should.equal(200);
+        JSON.parse(r.text).result.should.be.a("string");
+      });
+  }).timeout(600000);
+
+  it("Verifying ownership of NFT", async () => {
+    const myWallet = xrpl.Wallet.fromSeed(walletForSignatureVerification.seed);
+    let my_seq = 21404872;
+    const txJSON = {
+      Account: myWallet.address,
+      TransactionType: "Payment",
```

Comment:
As the example, instead of a payment, I'd recommend having them do a dummy `AccountSet` transaction because otherwise people may accidentally create a valid transaction which can send funds if someone intercepts it and actually executes it.